core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -75,16 +76,17 @@
      */
     public DefaultServiceRequestContext(
             ServiceConfig cfg, Channel ch, SessionProtocol sessionProtocol,
-            HttpMethod method, String path, PathMappingResult pathMappingResult, Object request,
+            PathMappingContext pathMappingContext, PathMappingResult pathMappingResult, Object request,
             @Nullable SSLSession sslSession) {
 
-        super(sessionProtocol, method,
-              path,
-              pathMappingResult.query(),
+        super(sessionProtocol,
+              requireNonNull(pathMappingContext, "pathMappingContext").method(), pathMappingContext.path(),
+              requireNonNull(pathMappingResult, "pathMappingResult").query(),
```

Comment:
Oops~ Sorry, I have checked only my local source and it was not there. Forget about this~